src/Notifier.php

```diff
@@ -23,9 +23,10 @@ class Notifier
     public function __construct($opt = [])
     {
         // TODO: test that projectId and projectKey exists
-        $this->opt = array_merge($opt, [
+        $this->opt = array_merge([
             'host' => 'api.airbrake.io',
-        ]);
+            'secure' => true,
```

Comment:
Can you please remove `secure => true` option and instead allow host to be in 2 forms:
- api.airbrake.io
- https://api.airbrake.io